<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Collecting</class>
 <widget class="QWidget" name="Collecting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始采集</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>步进电机通讯</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>500</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>终止采集</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>步长/°</string>
   </property>
  </widget>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>label</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>comboBox</zorder>
  <zorder>textBrowser</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>pressed()</signal>
   <receiver>Collecting</receiver>
   <slot>stopCollectButttonPush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Collecting</receiver>
   <slot>startCollect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Collecting</receiver>
   <slot>openClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>stopCollectButttonPush()</slot>
  <slot>startCollect()</slot>
  <slot>openClose()</slot>
 </slots>
</ui>
